docs/application/native/guides/text-input/tts.md

```diff
@@ -569,21 +578,20 @@ To add text:
   ```
 
 <a name="control"></a>
-## Controlling the Playback
+## Control playback
 
 To start, pause, and stop the playback:
 
 - To start synthesizing the text added in the queue and play the resulting sound data in sequence, use the `tts_play()` function.
 
   The state is changed to `TTS_STATE_PLAYING`, and the playback continues until you call the `tts_stop()` or the `tts_pause()` function.
 
-  If there is no text in the queue, the TTS waits for text to be added in the `TTS_STATE_PLAYING` state. In that case, when you add text, the TTS starts synthesizing and playing it immediately. The TTS state need not change to `TTS_STATE_READY` state before using the `tts_stop()` function.
+  If there is no text in the queue, TTS stays in the `TTS_STATE_PLAYING` state for text to be added. In that case, when text is added, TTS starts synthesizing and playing it immediately. There is no need to change the state to `TTS_STATE_READY` by using the `tts_stop()` function even when there is no text in the queue. It will continue to stay in the `TTS_STATE_PLAYING` state.
```

Comment:
Ref: Code snippets should be preceded by a “colon”.
```suggestion
  If there is no text in the queue, TTS stays in the `TTS_STATE_PLAYING` state for text to be added. In that case, when text is added, TTS starts synthesizing and playing it immediately. There is no need to change the state to `TTS_STATE_READY` by using the `tts_stop()` function even when there is no text in the queue. It will continue to stay in the `TTS_STATE_PLAYING` state:
```